examples/seismic/model.py

```diff
@@ -46,8 +46,8 @@ def initialize_damp(damp, nbl, spacing, mask=False):
         val = -val if mask else val
         eqs += [Inc(damp.subs({d: dim_r}), val/d.spacing)]
 
-    # TODO: Figure out why yask doesn't like it with dse/dle
-    Operator(eqs, name='initdamp', dse='noop', dle='noop')()
+    # TODO: Figure out why yask doesn't like it with default opt level
+    Operator(eqs, name='initdamp', opt='noop')()
```

Comment:
I thought we removed that 'noop' that was super slow.

Reply:
only from the builtins

Reply:
Maybe should remove here too?

Reply:
I **think** it still breaks yask if we remove it from here

Reply:
Oooh right yes probably, how long till yask is gone? :D